### ISSUE TITLE
Trying to add property or index to class - Execution Exception

### ISSUE THREAD

**Issue by cleverbug** [orientechnologies/orientdb#5461]
Hi,

as part of our test with OrientDB we've come across this weird behaviour:
- import a User class (with 8 cluster) with millions of vertexes
- try and add a property ( same thing happened with a multi field index, though)
- get an error during the execution

```

create property User.name STRING

Error: com.orientechnologies.orient.core.exception.OCommandExecutionException: Property 'User.name' already exists. Remove it before to retry.

orientdb {db=graphDb}> info class User                                                                              

CLASS 'User'

Super classes........: [V]
Default cluster......: user (id=11)
Supported clusters...: user(11), user_1(12), user_2(13), user_3(14), user_4(15), user_5(16), user_6(17), user_7(18)
Cluster selection....: round-robin
Oversize.............: 0.0

PROPERTIES
-------------------------------+-------------+-------------------------------+-----------+----------+----------+-----------+-----------+----------+
 NAME                          | TYPE        | LINKED TYPE/CLASS             | MANDATORY | READONLY | NOT NULL |    MIN    |    MAX    | COLLATE  |
-------------------------------+-------------+-------------------------------+-----------+----------+----------+-----------+-----------+----------+
 username                      | STRING      | null                          | false     | false    | false    |           |           | default  |
 email                         | STRING      | null                          | false     | false    | false    |           |           | default  |
 lastName                      | STRING      | null                          | false     | false    | false    |           |           | default  |
 uid                          | STRING      | null                          | false     | false    | false    |           |           | default  |
 firstName                     | STRING      | null                          | false     | false    | false    |           |           | default  |
-------------------------------+-------------+-------------------------------+-----------+----------+----------+-----------+-----------+----------+

INDEXES (1 altogether)
-------------------------------+----------------+
 NAME                          | PROPERTIES     |
-------------------------------+----------------+
 User.uid                     | uid           |
-------------------------------+----------------+
orientdb {db=graphDb}> select from user limit 10;

----+-----+------+---------------+---------------+--------+---------------------------+---------+--------+------+------+------------+----------+---------
#   |@RID |@CLASS|uid           |lastName       |username|email                      |firstName|birthday|userId|gender|birthdayYear|out_Friend|in_Friend
----+-----+------+---------------+---------------+--------+---------------------------+---------+--------+------+------+------------+----------+---------
0   |#11:0|User  |1234     |null           |        |email@email.com        |First |00-00   |1   |null  |0           |null      |null     
```

Same thing happened when I tried and create an index with:

```
create index User.name on User (firstName,lastName) FULLTEXT ENGINE LUCENE
```

**Comment by cleverbug** [comment]
Here's the error with the index:
```
create index User.name on User (firstName,lastName) FULLTEXT ENGINE LUCENE


Creating index...
Error: com.orientechnologies.orient.core.index.OIndexException: Index with name user.name already exists.
```

Although, looking at the log, it seems that the index was created:
```
2015-12-13 22:32:09:975 INFO  {db=graphDb} --> OK, indexed XXX items in YYY ms 
```

and the index does show:
```
select expand(indexes) from metadata:indexmanager

----+------+------+--------+---------+---------+---------+-----------+------------+------------------+--------------------------+-------------------------------
#   |@CLASS|mapRid|clusters|algorithm|name     |indexVers|type       |valueContain|metadata          |indexDefinitionClass      |indexDefinition                
----+------+------+--------+---------+---------+---------+-----------+------------+------------------+--------------------------+-------------------------------
....
2   |null  |null  |[8]     |LUCENE   |User.name|1        |FULLTEXT   |SBTREEBON...|null              |com.orientechnologies.o...|{className:User,indexDefinit...
...
----+------+------+--------+---------+---------+---------+-----------+------------+------------------+--------------------------+-------------------------------
```

Also, is it me or I'm not getting how you then query by the index?

```
SELECT FROM INDEX:User.name where key LUCENE 'Gazzetta*' limit 10    

----+------+-------------------------------+------------
#   |@CLASS|key                            |rid         
----+------+-------------------------------+------------
0   |null  |OCompositeKey{keys=[Gazzetta*]}|#16:47036837
```

Was kinda expecting to receive the User with the name like Gazzetta.

**Comment by cleverbug** [comment]
Any help with this?

I then tried and generate a normal fulltext hash index, but it is simply horribly performing ( ~20 items/sec)
```
create index User.firstNameIdx on User(firstName) FULLTEXT_HASH_INDEX
```

**Comment by luigidellaquila** [comment]
Hi @cleverbug 

which version are you using?

Thanks

Luigi

**Comment by cleverbug** [comment]
2.1.5 community downloaded from: http://orientdb.com/download.php?email=unknown@unknown.com&file=orientdb-community-2.1.5.tar.gz&os=linux

**Comment by luigidellaquila** [comment]
Hi @cleverbug 

is this issue still valid in 2.2.x?

Thanks

Luigi

**Comment by luigidellaquila** [comment]
Hi @cleverbug 

AFAIK this issue was fixed long time ago in 2.2.

I'm closing this ticket, please reopen if you still have problems with this

Thanks

Luigi
